docs/byte/1.md

```diff
@@ -344,26 +344,24 @@ $
 
 ## Questions and/or exercises
 
-As always, these questions are optional and only for fun.
```

Comment:
I'll put it back. Please note that I still disagree, though, as I don't really see the questions scaring someone away who read the entire page, and any opportunity to trim down the number of words will probably be useful.